recipes/sahi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sahi" %}
+{% set version = "0.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sahi-{{ version }}.tar.gz
+  sha256: 5a688f9782c78251669ddb3db3e280e2eb43a978102cec9aab87e5953070069f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - sahi=sahi.cli:app
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - fire
+    - py-opencv >=4.2.0.32
+    - pycocotools
```

Comment:
we can remove pycocotools dependency as its now optional: https://github.com/obss/sahi/pull/221